src/shader/codegen/msl.zig

```diff
@@ -531,7 +534,7 @@ fn emitStatement(msl: *Msl, inst_idx: InstIndex) error{OutOfMemory}!void {
         .discard => try msl.emitDiscard(),
         // .@"break" => try msl.emitBreak(),
         .@"continue" => try msl.writeAll("continue;\n"),
-        // .call => |inst| try msl.emitCall(inst),
+        .call => |inst| try msl.emitCall(inst),
```

Comment:
I think this should be removed, accidental dead-code.  Will be handled by emitExpr below?

Reply:
No. function calls can be appeared as both statements and expressions.

Reply:
Oops - sorry, I forgot the emitExpr case is not an `else`... emitCall doesn't include a `;` though so not sure how this works.  I think it should just be another case prong for line 541.

Reply:
Hmm, afaik function calls like `myFn();` are statements so `;` must be added in `emitStatement`